blocks/block-alignment-toolbar/index.js

```diff
@@ -25,6 +25,10 @@ const BLOCK_ALIGNMENTS_CONTROLS = {
 		icon: 'align-full-width',
 		title: __( 'Full width' ),
 	},
+	fixed: {
```

Comment:
Yes if we were to make a toggle for this it should just be the existing switch:

![screen shot 2017-06-22 at 19 57 02](https://user-images.githubusercontent.com/1204802/27448414-00cff9d4-5785-11e7-89e8-eb4fe580b2d8.png)

But I think it's okay to actually remove this advanced property for now to just get the PR merged, if that's simpler.

Alternately, I feel pretty strongly that the fixed background should be applied by default. And it should not be a `data-align` value. It should be something you can toggle _in addition_ to the image alignment.